### PR TITLE
Manager dashboard updates

### PR DESCRIPTION
**Pull request by aamishbaloch**
Changes include addition of model OrganizationUsersAttributes and the respective changes.

### CODE REVIEW

File: edx_solutions_organizations/models.py [hunk 1]
```diff
@@ -46,3 +92,34 @@ class Meta(object):
...
+        unique_together = ("user", "key")
+
+    @staticmethod
+    def get_all_attributes(user):
+        """
+        Gets all attributes for a given user
+
+        Returns: Set of (attributes type, value) pairs for each of the user's organizational attributes
+        """
+        return {(pref.key, pref.value) for pref in user.attributes.all()}
```
Comment (ihtram) [inline edx_solutions_organizations/models.py]: do we need to return active attributes only. please make sure

File: edx_solutions_organizations/utils.py [hunk 2]
```diff
@@ -7,18 +7,27 @@ def generate_key_for_field(data):
...
+    return name in [value['label'] for key, value in data.items()]
+
+
+def active_attributes(data):
+    """
+    Method used to filter only the active attributes of organizations
+    :param data: existing attributes with keys
+    :return: active attributes
+    """
+    return {key: value for key, value in data.items() if value['is_active'] == True}
```
Comment (ihtram) [inline edx_solutions_organizations/utils.py]: do we need this method? as this one is also defined in Organization model

**Review by ihtram — CHANGES_REQUESTED** [review]
a couple of changes required.

**Comment by aamishbaloch** [comment]
@ihtram Done with the changes. Please review.

**Review by ihtram — APPROVED** [review]
👍

**Pull request merged**